createst.py

```diff
@@ -143,6 +143,8 @@ def write_to_file(data):
         fp.write("# *** Add configuration here ***\n\n")
         if not args["strictcsums"]:
             fp.write("args:\n- -k none\n\n")
+        if args["min_version"]:
+            fp.write("requires:\n   min-version: %s\n\n" % args["min_version"])
```

Comment:
We should probably try to leave `requires` out of this write function. Since `requires` can have other things, there is a possibility of errored output here in case other features are added for the same `requires` section.

Reply:
Thanks for the feedback @inashivb ! I moved `requires` into a separate write function. Let me know if it's fine this way.